### PR TITLE
(COURSE): Added the marathi langauge courses markdown file

### PR DESCRIPTION
**Pull request by Atharva1723**
## What does this PR do?
Added free courses to the Marathi language playlists on youtube 

## For resources
YouTube

### Description
Added some good playlists links to programming languages such as html,css, angular,python etc in marathi language

### Why is this valuable (or not)?
Yes This is valuable as the marathi language courses was not added before and this is the new pr

### How do we know it's really free?
You can test the links 
### For book lists, is it a book? For course lists, is it a course? etc.
It is a markdown file for courses 
## Checklist:
- [ ✓] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ✓] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ✓] Include author(s) and platform where appropriate.
- [ ✓] Put lists in alphabetical order, correct spacing.
- [ ✓] Add needed indications (PDF, access notes, under construction).
- [✓ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Atharva1723** [comment]
@eshellman please review this

**Comment by Atharva1723** [comment]
@eshellman also please review the code of conduct

File: courses/free-courses-mr.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+* [Arduino Programming in Marathi](https://youtube.com/playlist?list=PL8yOrZ6_TOt5Y-ZG34wLePPZsDaZpatqD) - Asawari Shiposkar
+
+
+### <a id="c"></a>C
+
+* [C Language Tutorial Videos \| Krushna Pise \| MaRaTHi ProGrammer](https://youtube.com/playlist?list=PLWSZr_wlNWax9fqyNyt6Q3ADBgwLE2HvU) - MaRaTHi ProGrammer
+* [C Programming tutorials in Marathi](https://youtube.com/playlist?list=PLCx-k6Qe-qShOnyqpTckJd9qWflu0Ah5I) - M Computers
+
+
+### <a id="csharp"></a>C
```
Comment (3R1Dev) [inline courses/free-courses-mr.md:39]: Forgot to add the #

```suggestion
### <a id="csharp"></a>C#
```

File: courses/free-courses-mr.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+### Index
+
+* [Android](#android)
+* [Angular](#angular)
+* [Arduino](#arduino)
+* [C](#c)
+* [C#](#csharp)
```
Comment (3R1Dev) [inline courses/free-courses-mr.md:7]: Need to add C++ section

File: courses/free-courses-mr.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+* [Data Science](#datascience)
+* [Databases](#databases)
+* [HTML and CSS](#html-and-css)
+* [Java](#java)
+* [JavaScript](#javascript)
+* [PLC](#plc)
```
Comment (3R1Dev) [inline courses/free-courses-mr.md:13]: This is not used, better to remove it

```suggestion
```

**Review by 3R1Dev — CHANGES_REQUESTED** [review]
Just a few things to change.

**Comment by eshellman** [comment]
it is very difficult for the reviewers if you close the PR where the review comments have been made. Please use separate branches and separate PRs for the course list and the code of conduct, as different reviews are needed for each. Please remove code of conduct from this PR and create a fresh branch for it.

**Comment by Atharva1723** [comment]
@eshellman mb. I will make sure next time

**Comment by Atharva1723** [comment]
@eshellman  Is this okay?

**Comment by eshellman** [comment]
Thank you!

**Pull request merged**